server/mediaserver.go

```diff
@@ -663,3 +667,8 @@ func parseSegName(reqPath string) string {
 	}
 	return segName
 }
+
+func shouldStopStream(err error) bool {
+	trimmed := strings.TrimSpace(err.Error())
```

Comment:
Should we trim when reading the error body from the response? Otherwise LGTM!

Reply:
great point - will change

Reply:
@j0sh can you plz take another quick look at the commit with the refactor you suggested?